From 827a6d07f6f327cef9b364a8db8619fa1454fa28 Mon Sep 17 00:00:00 2001
From: Ian Lin <ian.lin-ee@infineon.com>
Date: Mon, 14 Mar 2022 20:55:22 -0500
Subject: [PATCH 117/179] revert "brcmfmac: Set timeout value when configuring
 power save"

This timeout value is the time STA stay awake before enter sleep.
Remove setting 2s and use fw default value 200ms
for better current consumption.
---
 .../net/wireless/broadcom/brcm80211/brcmfmac/cfg80211.c    | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/drivers/net/wireless/broadcom/brcm80211/brcmfmac/cfg80211.c b/drivers/net/wireless/broadcom/brcm80211/brcmfmac/cfg80211.c
index 55dd059446d3..6a5fd621182a 100644
--- a/drivers/net/wireless/broadcom/brcm80211/brcmfmac/cfg80211.c
+++ b/drivers/net/wireless/broadcom/brcm80211/brcmfmac/cfg80211.c
@@ -89,8 +89,6 @@
 
 #define BRCMF_ND_INFO_TIMEOUT		msecs_to_jiffies(2000)
 
-#define BRCMF_PS_MAX_TIMEOUT_MS		2000
-
 #define MGMT_AUTH_FRAME_DWELL_TIME	4000
 #define MGMT_AUTH_FRAME_WAIT_TIME	(MGMT_AUTH_FRAME_DWELL_TIME + 100)
 
@@ -3398,11 +3396,6 @@ brcmf_cfg80211_set_power_mgmt(struct wiphy *wiphy, struct net_device *ndev,
 			bphy_err(drvr, "error (%d)\n", err);
 	}
 
-	err = brcmf_fil_iovar_int_set(ifp, "pm2_sleep_ret",
-				min_t(u32, timeout, BRCMF_PS_MAX_TIMEOUT_MS));
-	if (err)
-		bphy_err(drvr, "Unable to set pm timeout, (%d)\n", err);
-
 done:
 	brcmf_dbg(TRACE, "Exit\n");
 	return err;
-- 
2.17.1

